cmd/flowrunner/testdata/flows/airtime_disabled_test.json

```diff
@@ -18,7 +18,6 @@
                     "created_on": "2018-07-06T12:30:04.123456789Z",
                     "input": "d2f852ec-7b4e-457f-ae7f-f8b243c49ff5",
                     "name": "Transfer",
-                    "node_uuid": "75656148-9e8b-4611-82c0-7ff4b55fb44a",
```

Comment:
previously needed to create the result, but now just looks up the node using `run.GetStep(event.StepUUID())`